docs/source/Support/bskReleaseNotes.rst

```diff
@@ -62,6 +62,11 @@ Version  |release|
   :ref:`scenarioRoboticArm` scenarios.
 - Fixed issue in which reading ``RWModel`` from RW message payloads when :ref:`vizInterface` was also
   imported would return a Swig Object instead of an enumerated integer.
+- Refactored the ``GaussMarkov`` class to enforce walk bounds and to deprecate 1.5x multiplier
```

Comment:
You say the 1.5x is deprecated.  This means it still works for a while?  I would be clearer and say the need for the 1.5x value has been removed.  Some modules were not setting this correctly to begin with.  Now setting the standard deviation results, always, in the expected behavior.  However, prior code setting this value will have to be updated to keep the same behavior as before.